mappings/net/minecraft/loot/context/LootContextParameterSet.mapping

```diff
@@ -0,0 +1,51 @@
+CLASS net/minecraft/class_8567 net/minecraft/loot/context/LootContextParameterSet
+	FIELD field_44883 world Lnet/minecraft/class_3218;
+	FIELD field_44884 parameters Ljava/util/Map;
+	FIELD field_44885 dynamicDrops Ljava/util/Map;
+	FIELD field_44886 luck F
+	METHOD <init> (Lnet/minecraft/class_3218;Ljava/util/Map;Ljava/util/Map;F)V
+		ARG 1 world
+		ARG 2 parameters
+		ARG 3 dynamicDrops
+		ARG 4 luck
+	METHOD method_51863 getWorld ()Lnet/minecraft/class_3218;
+	METHOD method_51864 addDynamicDrops (Lnet/minecraft/class_2960;Ljava/util/function/Consumer;)V
+		ARG 1 id
+		ARG 2 lootConsumer
+	METHOD method_51865 contains (Lnet/minecraft/class_169;)Z
+		ARG 1 parameter
+	METHOD method_51866 getLuck ()F
+	METHOD method_51867 get (Lnet/minecraft/class_169;)Ljava/lang/Object;
+		ARG 1 parameter
+	METHOD method_51868 (Lnet/minecraft/class_169;)Ljava/lang/Object;
+		ARG 1 parameter
+	METHOD method_51869 getOptional (Lnet/minecraft/class_169;)Ljava/lang/Object;
+		ARG 1 parameter
+	CLASS class_49 DynamicDrop
+		METHOD add (Ljava/util/function/Consumer;)V
+			ARG 1 lootConsumer
+	CLASS class_8568 Builder
+		FIELD field_44887 world Lnet/minecraft/class_3218;
+		FIELD field_44888 parameters Ljava/util/Map;
+		FIELD field_44889 dynamicDrops Ljava/util/Map;
+		FIELD field_44890 luck F
+		METHOD <init> (Lnet/minecraft/class_3218;)V
+			ARG 1 world
+		METHOD method_51870 getWorld ()Lnet/minecraft/class_3218;
+		METHOD method_51871 luck (F)Lnet/minecraft/class_8567$class_8568;
+			ARG 1 luck
+		METHOD method_51872 addDynamicDrop (Lnet/minecraft/class_2960;Lnet/minecraft/class_8567$class_49;)Lnet/minecraft/class_8567$class_8568;
+			ARG 1 id
+			ARG 2 dynamicDrop
+		METHOD method_51873 getParameter (Lnet/minecraft/class_169;)Ljava/lang/Object;
```

Comment:
I recommend dropping the `Parameter` suffix as these are already in a parameter set.

Reply:
Will do